source/DSCResources/DSC_CMSiteConfiguration/DSC_CMSiteConfiguration.psm1

```diff
@@ -0,0 +1,1232 @@
+        [UInt32]
+        $MaximumNumberOfSendingWakeupPacketBeforePausing,
+
+        [Parameter()]
+        [ValidateRange(0,100)]
+        [UInt32]
+        $SendingWakeupPacketBeforePausingWaitSec,
+
+        [Parameter()]
+        [ValidateRange(1,9)]
+        [UInt32]
+        $ThreadNumberOfSendingWakeupPacket,
+
+        [Parameter()]
+        [ValidateRange(0,60)]
+        [UInt32]
+        $SendingWakeupPacketTransmissionOffsetMins,
+
+        [Parameter()]
+        [String]
+        $ClientCertificateCustomStoreName,
+
+        [Parameter()]
+        [ValidateSet('FailSelectionAndSendErrorMessage','SelectCertificateWithLongestValidityPeriod')]
+        [String]
+        $TakeActionForMultipleCertificateMatchCriteria,
+
+        [Parameter()]
+        [ValidateSet('ClientAuthentication','CertificateSubjectContainsString','CertificateSubjectOrSanIncludesAttributes')]
+        [String]
+        $ClientCertificateSelectionCriteriaType,
+
+        [Parameter()]
+        [String]
+        $ClientCertificateSelectionCriteriaValue
+    )
+
+    Import-ConfigMgrPowerShellModule -SiteCode $SiteCode
+    Set-Location -Path "$($SiteCode):\"
+    $state = Get-TargetResource -SiteCode $SiteCode
+
+    try
+    {
+        $defaultValues = @(
+            'SiteCode','Comment','MaximumConcurrentSendingForAllSite','MaximumConcurrentSendingForPerSite',
+            'RetryNumberForConcurrentSending','ConcurrentSendingDelayBeforeRetryingMins','ThresholdOfSelectCollectionByDefault',
+            'ThresholdOfSelectCollectionMax','SiteSystemCollectionBehavior'
+        )
+
+        if ($PSBoundParameters.ContainsKey('ThresholdOfSelectCollectionByDefault') -or $PSBoundParameters.ContainsKey('ThresholdOfSelectCollectionMax'))
+        {
+            if ($PSBoundParameters.ContainsKey('ThresholdOfSelectCollectionByDefault'))
+            {
+                $collectionDefault = $ThresholdOfSelectCollectionByDefault
+            }
+            elseif ($state.ThresholdOfSelectCollectionByDefault)
+            {
+                $collectionDefault = $state.ThresholdOfSelectCollectionByDefault
+            }
+
+            if ($PSBoundParameters.ContainsKey('ThresholdOfSelectCollectionMax'))
+            {
+                $collectionMax = $ThresholdOfSelectCollectionMax
+            }
+            elseif ($state.ThresholdOfSelectCollectionMax)
+            {
+                $collectionMax = $state.ThresholdOfSelectCollectionMax
+            }
+
+            if (($collectionMax -ne 0) -and ($collectionMax -le $collectionDefault))
+            {
+                throw ($script:localizedData.CollectionError -f $collectionDefault, $collectionMax)
+            }
+        }
+
+        if ($state.SiteType -eq 'Primary')
+        {
+            $defaultValues += @('ClientCheckCertificateRevocationListForSiteSystem','UsePkiClientCertificate',
+            'RequireSigning','UseEncryption','EnableLowFreeSpaceAlert','EnableWakeOnLan','ClientCertificateCustomStoreName',
```

Comment:
line 587 has the additional stuff under the array indented. line 621 and all lines till the close ) should be indented from the array specifications for uniform look and know that its still part of the array. here and the one below as well

Reply:
Fixed in local repo.